tizen/src/constants.h

```diff
@@ -0,0 +1,96 @@
+#ifndef FLUTTER_PLUGIN_WORKMANAGER_CONSTANTS_H_
+#define FLUTTER_PLUGIN_WORKMANAGER_CONSTANTS_H_
+
+namespace constants {
+namespace methods {
```

Comment:
According to google c++ coding style, namespace should have empty lines up and down.